lib/swaggerUI.js

```diff
@@ -23,25 +23,42 @@ function staticServe(hyper, req) {
         if (reqPath === '/index.html') {
             // Rewrite the HTML to use a query string
             const cfg = hyper.config;
+            const css = `
+                .topbar-wrapper .link img {
+                    display: none;
+                }
+                .topbar-wrapper .link::after {
+                    content: "${cfg.ui_name}";
+                }
+                .swagger-ui .topbar .download-url-wrapper {
+                    display: none;
+                }
+                .swagger-ui .info li, .swagger-ui .info p, .swagger-ui .info table {
+                    font-size: 16px;
+                    line-height: 1.4em;
+                }
+                .scheme-container {
+                    display: none
+                }
+            `;
```

Comment:
Could you please add a comment to each of the CSS blocks explaining what exactly it is doing? Will be beneficial later for maintenance.

Reply:
Done